src/components/Banner/Banner.stories.tsx

```diff
@@ -30,14 +16,40 @@ export default {
 type Args = React.ComponentProps<typeof Banner> & {
   title: string;
   description: string;
-  action: any;
-};
-
-const action = <Button variant="secondary">See updates</Button>;
+  action: ReactNode;
+};
+
+const getAction = (variant?: Variant) => (
+  <Button status={variant} variant="secondary">
```

Comment:
Ah lol yeah, that is a bit odd. I'll change it to `status` since it's being passed into the `Button`. Btw thanks for reviewing these even though I hadn't rebased yet.. What a mess lol